test/db/formats/pe/virtrelocXP

```diff
@@ -7,6 +7,7 @@ RUN
 
 NAME=PE: corkami virtrelocXP.exe - entrypoint
 FILE=bins/pe/virtrelocXP.exe
+ARGS=-B 0x10000
```

Comment:
If we make rizin work well by default, this is not needed anymore.

Reply:
What is default?

For XP it's 0x00400000

Reply:
AFAIU that is the preferred value chosen by the linker, not the default value.